### DISCUSSION TITLE
Display actual numbers in "chartist"  graph type

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#884] [Ideas]
Right now does a graph of type chartist not show actual numbers (outside of f.e. the days of the month) for the graph itself.

As an example does the graph for "Total Stargazers" and "New stargazers per day" look something like this:
![stats-stargazers](https://user-images.githubusercontent.com/11576465/154045825-5f20b0c9-2d1e-48cc-8cbc-8054b443ef81.svg)

The graph as of now makes it difficult to put the values into perspective, since there are no real ways to know what each value means.
At most can I determine, that the amount of stargazers increased by 2 on the 7th of February and then by 2 more on 8th of February to finally increase again by one on 15th of February.

With actual numbers being displayed, i.e. on the left side, would it allow us to understand what those 4 sections actually represent relative to the graph itself.

This should be already doable considering that plugins such as Stock prices already display the numbers on the side.

**Comment by lowlighter** [comment]
Hi 👋 !

Initially there were no numbers on the side because each point was supposed to have its value displayed above it.
So it's definitely a bug, I'll open an issue 👍

**Reply by lowlighter** [reply]
Should have been fixed by #954
